packages/core/admin/admin/src/content-manager/components/DynamicTable/index.js

```diff
@@ -81,6 +81,7 @@ const DynamicTable = ({
       headers={tableHeaders}
       onConfirmDelete={onConfirmDelete}
       onConfirmDeleteAll={onConfirmDeleteAll}
+      onOnOpenDeleteAllModalTrackedEvent="willBulkDeleteEntries"
```

Comment:
typo, and I think this prop is a bit brutal no? Why not to call directly the event in the child?

Reply:
Because the entire deletion is handled by the DynamicTable component which is used in several places of the app